sources/packages/api/src/esdc-integration/models/esdc-integration.model.ts

```diff
@@ -0,0 +1,24 @@
+export const DATE_FORMAT = "YYYYMMDD";
```

Comment:
The date format variable name can be more specific. e.g esdc_date_format  or ecert_date_format

Reply:
As we are defining the constants in the specific folder, I believe this can be same.

Reply:
But what if esdc integration involves one more date format in future ?